src/pages/index.tsx

```diff
@@ -93,6 +93,15 @@ const Home = ({
   pageNumber,
   totalMessages
 }: Props) => {
+  const isDevEmail =
+    currentUser?.emailAddress &&
+    (currentUser?.emailAddress.includes("@example.com") || currentUser?.emailAddress.includes("@madetech.com"))
+
+  const shouldRedirectUser = isDevEmail || currentUser?.featureFlags?.httpsRedirect
```

Comment:
Why do we need this condition?

Reply:
I added it in when I was refactoring and the reason was because it breaks up the final conditional into something far more readable